diff --git a/node_modules/fido2-lib/lib/main.js b/node_modules/fido2-lib/lib/main.js
index 4c9fa29..3ac66e6 100644
--- a/node_modules/fido2-lib/lib/main.js
+++ b/node_modules/fido2-lib/lib/main.js
@@ -242,7 +242,7 @@ class Fido2Lib {
      */
 	async attestationResult(res, expected) {
 		expected.flags = factorToFlags(expected.factor, ["AT"]);
-		delete expected.factor;
+		// delete expected.factor;
 		return Fido2AttestationResult.create(res, expected);
 	}
 
@@ -343,7 +343,7 @@ class Fido2Lib {
      */
 	async assertionResult(res, expected) {
 		expected.flags = factorToFlags(expected.factor, []);
-		delete expected.factor;
+		// delete expected.factor;
 		return Fido2AssertionResult.create(res, expected);
 	}
 
diff --git a/node_modules/fido2-lib/lib/parser.js b/node_modules/fido2-lib/lib/parser.js
index b402562..6e67c7b 100644
--- a/node_modules/fido2-lib/lib/parser.js
+++ b/node_modules/fido2-lib/lib/parser.js
@@ -317,10 +317,10 @@ function parseAuthenticatorData(authnrDataArrayBuffer) {
 	}
 
 	// TODO: parse extensions
-	if (extensions) {
-		// extensionStart = offset
-		throw new Error("authenticator extensions not supported");
-	}
+	// if (extensions) {
+	// 	// extensionStart = offset
+	// 	throw new Error("authenticator extensions not supported");
+	// }
 
 	return ret;
 }
